integration_tests/test_single_node_simple_cli.py

```diff
@@ -354,11 +354,11 @@ def test05_custom_contract_execution(self):
             {
                 "name": "amount",
                 "value": {
-                    "int_value":1000000
+                    "long_value":1000000
                 }
             }
         ])
-        tx_hash_store_contract = cmd.run_contract(self.wallet_password, "wasm", wasm_path, param, 0.001, 50000000, self.wallet_anna)
+        tx_hash_store_contract = cmd.run_contract(self.wallet_password, "wasm", wasm_path, param, 0.01, 50000000, self.wallet_anna)
         print("Tx sent. Waiting for validation")
         time.sleep(self.tx_block_time * 3 + 1)
 
```

Comment:
It seems to need a test with small gas and classified as invalid tx
If feel hard, will add at https://github.com/hdac-io/friday/pull/112

Reply:
I will not touch this test.
And can we I a test case for lack of fees?

Reply:
Will do it but after fare system is merged.